Makefile

```diff
@@ -8,3 +8,11 @@ build:
 
 run:
 	docker run --env-file .env erema/wapi:0.1
+
+gen-mock:
+	mockgen  -source=src/service/auth/auth.go -destination=testutil/mock/auth/auth.go
```

Comment:
Think about mockgen version or add some comment where some developer can download that util